README.md

```diff
@@ -34,10 +37,59 @@ Note that there is currently no separate development environment for mixpanel, s
     yarn run lint
     ```
    
+##Working with the track user event endpoint:
+####Notes:
+- This endpoint optionally requires an authorization header, see below for each use case.
+- When adding your own events please be sure to update the Lexicon in Mixpanel with the meaning of your events and property names. (E.x. Terra-UI added an event called Workspace:Share with a property called numAdditions. Updating the Lexicon on Mixpanel gives a description of each of these letting folks know that a Workspace:Share event occurs when a workspace is shared with at least one new person and that the numAdditions is the count of the users that the workspace was shared with.)
+
+###Unregistered User Event Endpoint:
+This version of the endpoint does not require an authorization header. This is for use with users that are not signed into the application or have never registered and are considered anonymous users. This endpoint will then forward the event data for these users to Mixpanel identifying the user by a unique UUID4 id that your client application must provide to Bard.
+
+Example: 
+		
+    Request URL: https://terra-bard-dev.appspot.com/api/event
+    Request Method: POST
+    Request Payload: 
+                { 'event name', 
+                    properties: { hostname: 'url name', distinct_id: 'A UUID4 string'}
+                }
+
+###Registered User Event Endpoint:
+This version of the endpoint does require an authorization header. This is for use with users that are registered and currently signed into the application. This endpoint will verify user identification with Sam and forward the event data for these users to Mixpanel anonymizing the user identifiction before passing along to Mixpanel. Please note for a registered user you CANNOT send a user id, this must be sent implicitly by the user being signed in. This is to prevent fradulent user id's from being passed.
+
+Example:
+
+    Request URL: https://terra-bard-dev.appspot.com/api/event
+    Request Method: POST
+    authorization: Bearer token
+    Request Payload: 
+                { 'event name', 
+                    properties: { hostname: 'url name'}
+                }
```

Comment:
This is good information, but since it's endpoint-specific reference information, it might fit better in the API docs. (I think the doc generator has a special tag for producing example code).

For this document, perhaps we can frame it at a higher level, and walk through how to solve various problems end to end. For example, "Tracking events for logged in users", "Tracking and identifying events for anonymous users", and "Establishing user profiles and emailing users".

That gives us a place to talk about intended usage patterns, which (IMO) is a mark of effective technical documentation. For example, things like: Since the syncProfile call is idempotent, it's acceptable to call repeatedly (e.g. on every login from the front end) in order to simplify the process of creating profiles for existing users.

Reply:
Thank you very much for the feedback, I think I've been conflating docs with this readme and so that clarification is appreciated.